### DISCUSSION TITLE
Death Animations for NPCs.

### DISCUSSION THREAD

**Discussion by HMxMisfit** [AscensionGameDev/Intersect-Engine#2098] [Ideas]
I would like to be able to play a death animation for NPCs when they die. 
This video is an example of a death animation that would play. 

https://github.com/AscensionGameDev/Intersect-Engine/assets/150658107/4f541647-2e5b-46ba-9224-172b15937192

There are two ways that I can think of that would work for this:
1. Putting _death in the Json file that handles sprites. Ie. Sprite_death.
2. Putting an option in the NPC Editor to play an On-Death Animation. This would check the map tile the NPC was on and play the animation accordingly.

**Comment by PyroTech03** [comment]
I'd love to see this as part of giving us "Death boxes" or corpses to loot instead of dropping everything to the ground.

**Comment by WeylonSantana** [comment]
duplicate of mine https://github.com/AscensionGameDev/Intersect-Engine/discussions/1497
